### ISSUE TITLE
New rule on VPC_VPN_2_TUNNELS_UP

### ISSUE THREAD

**Issue by shikharj05** [awslabs/aws-config-rules#228]
Kindly provide comments around Gherkin if any.
```
Description:
  Checks that both the VPN tunnels provided by AWS Site-to-Site VPN are in UP status. The rule returns NON_COMPLIANT if one or both the tunnels are reporting status as DOWN.
Trigger:
  Configuration Changes
Reports on:
  AWS::EC2::VPNConnection
Rule Parameters:
  None
Scenarios:
  Scenario: 1
     Given: At least one VPN tunnel provided by AWS Site-to-Site VPN connection has status as DOWN.
      Then: Return NON_COMPLIANT with annnotation 'This AWS VPN Connection has at least one VPN tunnel down with the "statusMessage"(from configuration item)'.
  Scenario: 2
     Given: Both the tunnels provided by AWS Site-to-Site VPN Connection have status as UP.
      Then: Return COMPLIANT
```

**Comment by jongogogo** [comment]
Look good. Just, is it not supposed to be `status` instead of `statusMessage` in the CI vgwTelemetry ?

**Comment by shikharj05** [comment]
@jongogogo 'status' tells us if the tunnel is UP or DOWN and statusMessage gives us the reason for tunnel being DOWN; for example 'IPSEC IS DOWN'. So we can use the statusMessage in annotation and state to determine compliance.

**Comment by jongogogo** [comment]
I see, make sense. Good to go :)
